edudoc/delegate-crash-course/delegate_crash_course.md

```diff
@@ -335,39 +335,40 @@ It is important to handle these requests as expeditiously as you can. You may ei
+To place an order, submit the [application](https://docs.google.com/forms/d/e/1FAIpQLSc594bJA55CahKr7F8P-oHK44tn0cZ9NBZst0H5udwKnmh9Og/viewform) with the equipment you want, the shipping address you want it sent to, and the person or organization who will be invoiced for the equipment. After placing the order, a member of the WCA Marketing Team will contact you to confirm the order contents and the recipient's mailing address, and then place the order with Speed Stacks on your behalf. Once the order ships, You will receive an invoice from the WFC, which you can pay via Stripe, PayPal, or bank transfer.
+
+Once you receive your equipment, and before using it in a competition, you should check that everything is in working order. If there are any issues, respond in the email thread with WMT. Once equiment has been used in a competition, it is harder to get a replacement.
+
+Finally, respond to the email thread with WMT to confirm that you have received the equipment and that everything is in working order, so that they can close out your order.
 
 ## Recommending a Delegate
 
-_(updated on January 30, 2020)_
+_(updated on September 11, 2023)_
 
 For becoming a WCA Delegate, the following steps should be taken:
 
-- Recommend a person to the Senior Delegate with a reason why this person should be a Delegate and why the region the person is located needs a Delegate
-- If the Senior Delegate agrees, then the Senior Delegate will fill out the application form.
-- If the Board agrees, then the following steps should be taken:
-  - The WCA Executive Assistants Team will create a G Suite account for the new Junior Delegate
-  - The Junior Delegate needs to create a WCA user account, link it to their WCA profile, and upload a nice photo of themself
-  - The Senior Delegate will update the Junior Delegate's status on the website
-  - The Junior Delegate must agree to the Code of Ethics and disclose any potential conflicts of interest
-  - The Junior Delegate needs to send a short self-introduction to the Board (and the Delegates group if they want to)
-  - The new Junior Delegate has to fill up their Delegate details in [their profile](https://www.worldcubeassociation.org/profile/edit).
-- After all the above steps are taken, we officially have a new Junior Delegate!
+1. Recommend a person to your Senior and Regional Delegate with a reason why this person should be a Delegate and why the region needs a Delegate
+1. If the Senior Delegate and Regional Delegate agree, then the Senior Delegate will fill out the application form and submit it to the WCA Board
```

Comment:
You put 1. everywhere on accident, but the numbers should keep going up

Reply:
This is intentional. In markdown, numbers on the rendered document will be in ascending order. You can see that it behaves correctly in the rich diff:

![image](https://github.com/thewca/wca-documents/assets/33607815/0f99dd8b-e696-4466-b7b5-7323d5eac0dd)

Repeatedly using `1.` is a common pattern so that if you add or remove a step, you don't have to re-number all the lines that come after it.

Reply:
Dang, I didn't know that, thanks for letting me know!